### ISSUE TITLE
train set and validation set

### ISSUE THREAD

**Issue by tlhoerd** [ClmRmb/SEN12-FLOOD#1]
How can I divide the training set and the validation set? It seems that the official doesn't give a division scheme.

**Comment by PradeepYLN** [comment]
@tlhoerd Hope you have the code with you, code is not accessible now. It would be nice if you can share it with us
